From 119bb79f4fd0e35d6db2a7046245b4bf7fbb4a32 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Tue, 6 Mar 2012 10:16:46 +0800
Subject: ENGR00176147-3: usb: change data types for wakeup_event

In order to know which wakeup event occurs for usb wakeup

Signed-off-by: Peter Chen <peter.chen@freescale.com>
---
 include/linux/fsl_devices.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/fsl_devices.h b/include/linux/fsl_devices.h
index dd95492..5b701a8 100644
--- a/include/linux/fsl_devices.h
+++ b/include/linux/fsl_devices.h
@@ -139,7 +139,7 @@ struct fsl_usb2_platform_data {
 	unsigned	ahb_burst_mode:3;
 	unsigned	lowpower:1;
 	unsigned	irq_delay:1;
-	unsigned	wakeup_event:1;
+	enum usb_wakeup_event	wakeup_event;
 	u32		pmflags;	/* PM from otg or system */
 
 	/* register save area for suspend/resume */
-- 
1.7.9.5

